### ISSUE TITLE
Running multiple instances causes eventual fs error

### ISSUE THREAD

**Issue by joncursi** [meteor/meteor#7032]
Prior to upgrading to 1.3, i was able top run multiple meteor apps locally while developing. I.e. a web front-end client on port 3000, a cordova ios simulator on port 4000, and a back-end worker on 9000, etc.

Now, with 1.3, when I boot up multiple meteor instances of the same app, on different ports, I get this error after a few minutes of the apps running in parallel:

```
                                              
/Users/jcursi/.meteor/packages/templating/.1.1.9.1aa2v5e++os+web.browser+web.cordova/plugin.compileTemplatesBatch.os/npm/node_modules/meteor/promise/node_modules/meteor-promise/promise_server.js:116
      throw error;
            ^
Error: UNKNOWN, open '/Users/jcursi/Sites/joncursi/redbird/node_modules/inline-style-prefixer/lib/plugins/sizing.js'
```

Seems like Meteor wants to lock some files, blocking other instances from reading?

I am running 1.3.2.4 with

```
{
  npm: '2.14.22',
  ares: '1.9.0-DEV',
  http_parser: '1.2',
  modules: '11',
  node: '0.10.43',
  openssl: '1.0.1s',
  uv: '0.10.36',
  v8: '3.14.5.9',
  zlib: '1.2.8' 
}
```

**Comment by abernix** [comment]
The promise errors are pretty ambiguous right now and it's probably unrelated.

* What version of OS X are you using?
* Do your projects have a large number of files?  Try including the output of this run from the project roots:

        find . -type f | cut -d / -f 2 | sort | uniq -c | sort -rn | head -5

* Is there any more to that stack trace?

* What makes you say this?:
> Seems like Meteor wants to lock some files, blocking other instances from reading?

I haven't run into this myself and I have anywhere from 3-6 running (for at least, say, 15 minutes minimum?) throughout my day.

Would be great if you could reproduce this with some barebones apps that you could share!

**Comment by joncursi** [comment]
Thanks for your reply.

* OSX: 10.11.4 (15E65) El Capitan
* Command output of the project in question:

  ```
118901 node_modules
4307 .meteor
2420 .git
 914 node_shrinkwrap
 101 imports
  ```

Lots of package dependencies being pulled in from NPM.

**Comment by abernix** [comment]
So, unless you can reproduce this by running several *smaller* projects, I'm going to mark this as a duplicate of #4660 and possibly #6952.  Closing this in favor of those – so click the thumbs-up :+1: on them, if you would!
